selfdrive/ui/translations/main_zh-CHS.ts

```diff
@@ -124,7 +124,7 @@
     </message>
     <message>
         <source>N/A</source>
-        <translation>N/A</translation>
+        <translation>空</translation>
```

Comment:
Can we keep this PR to only adding new translations?

Reply:
OK